digipost-vilkaar-privat-engelsk.md

```diff
@@ -5,13 +5,13 @@ Digipost, and by accepting the terms you also give permission to process
 personal data about yourself. You must therefore review the terms and
 conditions carefully and assess whether you accept these before you start
 using Digipost. If you have any questions about the terms and conditions, we
-recommend that you contact Norway Post by email at
+recommend that you contact Posten Bring by email at
 [kundeservice@digipost.no](mailto:kundeservice@digipost.no), and
-await Norway Post’s response before accepting the terms and conditions and
+await Posten Bring’s response before accepting the terms and conditions and
 using Digipost.
 
 Below are the terms and conditions governing the use of Digipost (“Terms”), a
-service provided by Posten Norge AS (“Norway Post”), co. reg. no. 984 661 185.
+service provided by Posten Bring AS, co. reg. no. 984 661 185.
```

Comment:
Ettersom alle instanser av "Norway Post" er blitt erstattet med "Posten Bring" så har jeg fjernet teksten ("Norway Post") ettersom det ikke blir referert til lenger for engelsk navn.

Reply:
Enig.